### PR TITLE
Rebalences constructs to make wraith more atractive and jug less uncounterable

### PR DESCRIPTION
**Pull request by Supermichael777**
Because while I'm re-balancing jug I may as well make some tweaks to the rest.

Details:
Juggernaut base reflect chance cut from 80 to 40 because reflect is ass for the unstunable death machine. Juggernaut speed increased from 3 to 2 to help them actually catch people.

Nerfed artificer speed from 0 to 1. reduced the number of mm projectiles they get to 4.

Buffed wraiths from 75 health to 100. This makes them equally durable to humans, making them more viable.

Homogenized harvester attacks to 1 damage because if you need to use that your doing it wrong and rng is terrible.

These numbers subject to debate and revision.

### CODE REVIEW

**Comment by bgobandit** [comment]
this conflicts with https://github.com/tgstation/-tg-station/pull/11437, or will

**Comment by RemieRichards** [comment]
Don't leave comments like that.

**Comment by Supermichael777** [comment]
@bgobandit I can add that line on my end.

**Comment by Supermichael777** [comment]
TRAVIIIIIIIIIS

**Comment by Nanotransistor** [comment]
I dont see juggs are being uncounterable. They move so slow its easy to just outrun them as you shoot at them. Another thing is the actual reflection, that reflects shots at an angle, meaning you are really unlikely to get hit by your own lasers.

A better way would be to make every shot be reflected in a straight line(100% reflection) and make the jugg take damage even after it has reflected it.

File: code/modules/mob/living/simple_animal/constructs.dm [hunk 1]
```diff
@@ -16,6 +16,7 @@
 	attack_sound = 'sound/weapons/punch1.ogg'
 	atmos_requirements = list("min_oxy" = 0, "max_oxy" = 0, "min_tox" = 0, "max_tox" = 0, "min_co2" = 0, "max_co2" = 0, "min_n2" = 0, "max_n2" = 0)
 	minbodytemp = 0
+	healable = 0
```
Comment (Anonus) [inline code/modules/mob/living/simple_animal/constructs.dm]: EXCUSE ME, WHAT IS THIS

**Comment by Anonus** [comment]
Harvesters probably shouldn't get nerfed as they're literally a round-end thing only and weak enough that they don't need nerfs at all.

**Comment by MMMiracles** [comment]
Why nerf the reflect chance for lasers? Jugs give security the perfect excuse to pull out the lethal shotty slugs. Jugga get destroyed against slugs, so they already got a decent counter.

**Comment by Robustin** [comment]
I love cult and do not want to see them weakened.

However, reflect chance is one of my least favorite aspects of it.

Turn reflect into absorb/block and then throw in the wraith buffs to compensate, getting lethal slugs isn't terribly easy when the stations going to hell and you don't have access to the only autolathe.

**Comment by PKPenguin321** [comment]
:-1: for harvester nerf, they're roundend constructs and it wouldnt matter if they had 1000 damage per hit because the station's dead anyways, just leave them as is
:-1: for artificer nerf, why are you nerfing the construct that nobody wants to play and people already ghost out of to ragequit
:-1: for juggernaut "nerf," it's more of a buff
:unamused: by that wraith buff, oh boy 25 extra health wow this will sure change the meta (it wont)

**Comment by Supermichael777** [comment]
alright feedback has been overwhelmingly negative and merge conflicts. closing this untill i get a beter idea of the changes people want.

**Pull request closed without merging**